docs/api/browser-window.md

```diff
@@ -61,6 +61,7 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
   key is pressed.
 * `enable-larger-than-screen` Boolean - Enable the window to be resized larger
   than screen.
+* `background-color` Integer - Window's background color.
```

Comment:
Could this be expanded to indicate how the colour is coded into an integer value?